source/dub/project.d

```diff
@@ -1761,38 +1771,33 @@ final class SelectedVersions {
 		m_dirty = true;
 	}
 
-	/// Selects a certain version for a specific package.
-	void selectVersion(string package_id, Version version_)
+	/// Selects the exact dependency for a specific package.
+	void selectVersion(string package_id, Dependency d)
```

Comment:
I was trying to remove any use of `Dependency` from the API because it's almost always wrong. Why re-introduce it?